app/assets/javascripts/observation_form_map.js

```diff
@@ -0,0 +1,96 @@
+// Observation Form Map - Lat/Long/Alt Helper
+
+// ./observer/create_observation
+if ( $('#observationFormMap').length ) {
+
+    (function() {
+        var searchInput = $('#observation_place_name'),
+            latInput = $('#observation_lat'),
+            lngInput = $('#observation_long');
+
+        // init map object
+        var map = new google.maps.Map(document.getElementById('observationFormMap'), {
+            zoom: 1,
+            center: { lat: 8.38321792370903, lng: -66.56186083652352 }
```

Comment:
I'm debating where to take my next trip! Honestly, it probably doesn't need to be so specific... just felt it setup for a nice vantage point.